react-intl/index.d.ts

```diff
@@ -34,14 +34,18 @@ declare namespace ReactIntl {
         formats?: any;
     }
 
-    interface InjectedIntl {
```

Comment:
We actually lint to use `interface` instead of `type =`. (This package has no `tslint.json`, so it isn't linted.) And the rest of the file uses 4-space indents.